packages/compiler/templates/emitter-ts/test/test-host.ts

```diff
@@ -0,0 +1,47 @@
+import { Diagnostic, resolvePath } from "@typespec/compiler";
+import {
+  createTestHost,
+  createTestWrapper,
+  expectDiagnosticEmpty,
+} from "@typespec/compiler/testing";
+import { TestLibrary } from "../src/testing/index.js";
+
+export async function createMyTestHost() {
```

Comment:
I find these names to be a little awkward with "my" in them, could maybe template in the emitter name in place of "my"?

Reply:
Yeah that would be great but I think our template engine right now doesn’t have this capability to change casing. I can do a follow up pr that does